tests/Kernel/ContentLanguageSwitcherTest.php

```diff
@@ -111,8 +111,8 @@ public function testLanguageSwitcherRendering(): void {
    *   The label of the language.
    */
   protected function assertUnavailableLanguage(Crawler $crawler, string $expected): void {
-    $actual = $crawler->filter('.ecl-lang-select-page .ecl-u-type-strike')->text();
-    $this->assertEquals($expected, trim($actual));
+    $actual = $crawler->filter('div.ecl-u-bg-grey-5.ecl-u-pv-m div.ecl-container div.ecl-message--warning')->text();
+    $this->assertContains($expected, trim($actual));
```

Comment:
I suppose that it would be nice if we insure that exist an empty element for etranslation.

Reply:
But there is no empty element ATM, we only provide empty blocks that then can be used to add the etranslation element (or whatever else is needed)